misc/python/materialize/cli/mzcompose.py

```diff
@@ -142,6 +144,19 @@ def main(argv: list[str]) -> None:
 
 def load_composition(args: argparse.Namespace) -> Composition:
     """Loads the composition specified by the command-line arguments."""
+    if not args.ignore_docker_version:
+        local_version = Version.parse(
+            spawn.capture(["docker", "--version"])
+            .removeprefix("Docker version ")
+            .split(", ")[0]
+        )
+        ci_version = Version.parse("24.0.5")
+        if local_version < ci_version:
+            print(
+                f"Your Docker version is {local_version} while the version used in CI is {ci_version}, please upgrade your local Docker version to prevent unexpected breakages. If you believe this is a mistake, contact the QA team. While not recommended, --ignore-docker-version can be used to ignore this version check."
+            )
+            sys.exit(1)
```

Comment:
Wait, this is the Docker version. Good to check too, but I think it's the Docker _Compose_ version that specifically bit Gus most recently.

Reply:
It is my understanding that nowadays docker compose is part of docker and not a separate tool anymore. I don't have a `docker compose --version` https://docs.docker.com/compose/migrate/

Reply:
Try `docker compose version` (without `--`)

Reply:
```
nrainer@mz-nrainer mz-repo % docker --version
Docker version 24.0.6, build ed223bc
nrainer@mz-nrainer mz-repo % docker compose version
Docker Compose version v2.22.0-desktop.2
```

Reply:
> It is my understanding that nowadays docker compose is part of docker and not a separate tool anymore. I don't have a docker compose --version https://docs.docker.com/compose/migrate/

It's confusing, but it's both, AFAIK. It ships with Docker by default, but you can also install your own version of Docker Compose outside of Docker.